Changelog.md

```diff
@@ -54,7 +54,7 @@
 1. Include overrides for actor and light
     * [Pull request 669](https://osrf-migration.github.io/sdformat-gh-pages/#!/osrf/sdformat/pull-requests/669)
 
-1. Add functionality to generate aggregated SDFormat descriptions via CMake.
+1. Add functionality to generate aggregated libsdformat descriptions via CMake.
```

Comment:
This is referring to the spec, so it should be SDFormat

Reply:
Done.